requirements/development.in

```diff
@@ -14,6 +14,8 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 #
+-r base.txt
```

Comment:
Why is this needed? As development dependencies might be incompatible with the pinnings in `base.txt`, this might cause a clash.